azure-pipelines.yml

```diff
@@ -545,10 +545,27 @@ jobs:
              --location \
              -d "{\"title\": \"update compat versions for $(release_tag)\", \"head\": \"$BRANCH\", \"base\": \"master\"}" \
              https://api.github.com/repos/digital-asset/daml/pulls
+        setvar "branch" "$BRANCH"
+      name: out
+
     - template: ci/tell-slack-failed.yml
       parameters:
         trigger_sha: '$(trigger_sha)'
 
+  - job: compat_versions_pr_trigger_daily
```

Comment:
This feels a bit off. Shouldn’t we consider this a blocking check and do something like using a special format for the update PR and match on that on every PR?

Reply:
I'm not sure I understand what you mean here. This will trigger an independent build that reports its status to the branch and thus blocks the PR, exactly as if you'd triggered it through the web UI.

Reply:
Ah I didn’t know independent builds were considered blocking. Then this seems reasonable :+1:

Reply:
My understanding is that:
- We have a set number of builds that GH will wait for specifically; however
- All builds reported for the "tip" commit of a PR are part of the PR status;
- Any build reported as anything else than succeeded (i.e. pending or failed) blocks the PR from being merged.

I'll keep an eye on this when we do the next snapshot.